src/test/resources/mockdata/vendors/active_inactive_access_providers.json

```diff
@@ -0,0 +1,347 @@
+{
```

Comment:
Note that there's another PR open for updating these data files to have camelCase fields... Please coordinate accordingly.  https://github.com/folio-org/mod-orders/pull/112

Reply:
whoops, these are vendor records.  disregard my earlier comment